tests/playwright/assets/breakpoints.ts

```diff
@@ -33,6 +33,29 @@ export default class {
 		return [ 'mobile', 'tablet', 'desktop' ];
 	}
 
+	async saveOrUpdate( editor: EditorPage, toReload = false ) {
+		const hasTopBar: boolean = await editor.hasTopBar();
+		if ( hasTopBar ) {
+			const saveButton = "//button[text()='Save Changes']";
```

Comment:
Maybe use `EditorSelectors` for selectors like 'Save Changes'?

Reply:
I've moved the locator for 'Save Changes' and other locators to the EditorSelectors file. However, I’m using XPath for this button instead of getByRole. This approach is necessary because the button might not be rendered on the UI, which prevents the use of getByRole with this.page.evaluate.
If you have other ideas i can change it.

Reply:
@Svitlana-Dykun 
For clarification, why does it not always get rendered on the UI?